sabayon-brokenlinks

```diff
@@ -0,0 +1,457 @@
+#!/bin/bash
+# Authors: Sabayon Team 2019
+
+set -e
+
+# Accepted values: x86_64, armv7l
+export SAB_ARCH="${SAB_ARCH:-}"
+export DRY_RUN=${DRY_RUN:-0}
+export DEBUG="${DEBUG:-}"
+export FORCE_MANUAL="${FORCE_MANUAL:-0}"
+export GENTOO_TOOL="${GENTOO_TOOL:-0}"
```

Comment:
Wouldn't be USE_EQUO more consistent here as we use it in other parts?

Reply:
Ok. I copy that.